src/components/environments-summary/environment-ingress.js

```diff
@@ -1,40 +1,98 @@
+import { faExclamationCircle } from '@fortawesome/free-solid-svg-icons';
 
 import usePollComponents from './use-poll-components';
 
 const URL_VAR_NAME = 'RADIX_PUBLIC_DOMAIN_NAME';
-const MAX_DISPLAY_NR_COMPONENT = 3;
+const MAX_DISPLAY_NR_COMPONENT = 4;
+
+const outdatedOrFailedComponent = (component) => {
+  if (component.status === 'Outdated') {
+    return (
+      <span
+        class="env-summary-image-outdated-warning"
+        title="Component is running an outdated image"
+      >
+        <FontAwesomeIcon icon={faExclamationCircle} />
+      </span>
+    );
+  }
+  if (component.status === 'Failing') {
+    return (
+      <span
+        class="env-summary-image-outdated-warning"
+        title="Component is failing"
+      >
+        <FontAwesomeIcon icon={faExclamationCircle} />
+      </span>
+    );
+  }
+};
 
 const EnvironmentIngress = ({ appName, deploymentName }) => {
   const [componentsPollState] = usePollComponents(appName, deploymentName);
+
   let components = [];
   if (componentsPollState && componentsPollState.data) {
-    components = componentsPollState.data.filter(
-      (component) => component.variables[URL_VAR_NAME]
+    components = componentsPollState.data;
+  }
+
+  let publicComponents = [];
+  let passiveComponents = [];
+  if (components) {
+    publicComponents = components.filter(
+      (comp) => comp.variables[URL_VAR_NAME]
+    );
+    passiveComponents = components.filter(
+      (comp) => !comp.variables[URL_VAR_NAME]
     );
   }
 
   if (components.length <= 0) {
     return <div />;
   }
-  const tooManyPublicComponents = components.length > MAX_DISPLAY_NR_COMPONENT;
+  const tooManyPublicComponents =
+    publicComponents.length > MAX_DISPLAY_NR_COMPONENT;
+
+  const tooManyPassiveComponents =
+    passiveComponents.length > MAX_DISPLAY_NR_COMPONENT;
+
   if (tooManyPublicComponents) {
-    components = components.slice(0, MAX_DISPLAY_NR_COMPONENT);
+    publicComponents = publicComponents.slice(0, MAX_DISPLAY_NR_COMPONENT);
+  }
+
+  if (tooManyPassiveComponents) {
+    passiveComponents = passiveComponents.slice(0, MAX_DISPLAY_NR_COMPONENT);
   }
 
   return (
     <div>
       <ul>
-        {components.map((component) => (
-          <li key={component.name}>
-            <a href={`https://${component.variables[URL_VAR_NAME]}`}>
-              {component.name} <FontAwesomeIcon icon={faLink} size="lg" />
-            </a>
+        {publicComponents.map((component) => (
+          <li key={component.name} class="env-summary-component-list">
+            <div class="env-summary-ingress-container">
+              <a href={`https://${component.variables[URL_VAR_NAME]}`}>
+                {component.name} <FontAwesomeIcon icon={faLink} size="lg" />
+              </a>
+              {outdatedOrFailedComponent(component)}
+            </div>
           </li>
         ))}
-        {tooManyPublicComponents && <li>...</li>}
+        {passiveComponents.map(
+          (component) =>
```

Comment:
Ref comment about using one slice would remove the need for two <slice>.map(...) iterations